indra/llcommon/llstring.h

```diff
@@ -1595,6 +1596,20 @@ void LLStringUtilBase<T>::replaceTabsWithSpaces( string_type& str, size_type spa
 	str = out_str;
 }
 
+//static
+template<class T>
+void LLStringUtilBase<T>::capitalize(string_type& str)
+{
+	if (str.size())
+	{
+		auto last = str[0] = toupper(str[0]);
```

Comment:
Not sure if this is the right approach for foreign language characters - I think there is a more robust method in the std library - Nat would know for sure.

Reply:
> Not sure if this is the right approach for foreign language characters - I think there is a more robust method in the std library - Nat would know for sure.

I'm sure you're right! But for now we could commit this as is, and replace the method implementation any time later